src/Data/Dictionary/DictionaryItem.php

```diff
@@ -0,0 +1,117 @@
+<?php
+declare(strict_types=1);
+namespace LORIS\Data\Dictionary;
+
+use \LORIS\Data\Scope;
+use \LORIS\Data\Type;
+use \LORIS\Data\Cardinality;
+
+/**
+ * A DictionaryItem represents a description of a type of data
+ * managed by LORIS.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class DictionaryItem implements \LORIS\StudyEntities\AccessibleResource
```

Comment:
abstract?

Reply:
Nervermind. I just looked at the "Main" PR and the intention is not for module to create nes classes of DictionnaryItem but to instanciate objects.

Reply:
yes, but they can extend it to customize the accessibility if need be.